spec/hopper_spec.rb

```diff
@@ -340,6 +340,7 @@ def handle_object_created(_event_type, _event, _source); end
       expect(described_class.queue.message_count).to be_zero
     end
 
+    # rubocop:disable RSpec/ExampleLength
```

Comment:
that is a dumb rule - we could configure to ignore that one

Reply:
I agree !

Reply:
done